kyuubi-server/src/main/scala/org/apache/kyuubi/server/RestFrontendService.scala

```diff
@@ -33,7 +33,7 @@ import org.apache.kyuubi.service.{AbstractFrontendService, Serverable, Service}
  * A frontend service based on RESTful api via HTTP protocol.
  * Note: Currently, it only be used in the Kyuubi Server side.
  */
-private[server] class RestFrontendService(override val serverable: Serverable)
+class RestFrontendService(override val serverable: Serverable)
```

Comment:
Maybe `RESTfulFrontendService` is a better name? cc @yanghua

Reply:
sounds good. +1